src/components/accordion/Accordion.spec.jsx

```diff
@@ -163,4 +163,83 @@ describe('<Accordion>', () => {
     // hostNodes returns html elements and skip react components
     expect(accordion.find('[aria-expanded=true]').hostNodes()).toHaveLength(2);
   });
+
+  it('expands controlled items when expanded is type of array', () => {
+    const accordionIds = ['id-1', 'id-2', 'id-3'];
+    const accordion = mount(
+      <Accordion expanded={accordionIds} onChange={() => undefined}>
+        {accordionIds.map(id => (
+          <AccordionItem title={id} id={id} key={id}>
+            Accordion Item Description
+          </AccordionItem>
+        ))}
+      </Accordion>
+    );
+
+    expect(accordion.find('[aria-labelledby]').hostNodes()).toHaveLength(
+      accordionIds.length
+    );
+    expect(accordion.find('[aria-expanded=true]').hostNodes()).toHaveLength(3);
+  });
+
+  it('expands controlled item when expanded is type of string', () => {
+    const accordionIds = ['id-1', 'id-2', 'id-3'];
+    const accordion = mount(
+      <Accordion expanded={accordionIds[0]} onChange={noop => noop}>
+        {accordionIds.map(id => (
+          <AccordionItem title={id} id={id} key={id}>
+            Accordion Item Description
+          </AccordionItem>
+        ))}
+      </Accordion>
+    );
+
+    expect(accordion.find('[aria-labelledby]').hostNodes()).toHaveLength(
+      accordionIds.length
+    );
+    expect(accordion.find('[aria-expanded=true]').hostNodes()).toHaveLength(1);
+  });
+
+  it('expands controlled item when expanded is type of string', () => {
```

Comment:
my duplicate 🙈  could you remove it for me if you are planning merge it to master?

Reply:
oopsie, removed :)